service/gcs/core/gcs/gcs.go

```diff
@@ -117,14 +116,18 @@ func (e *containerCacheEntry) RemoveMappedDirectory(dir prot.MappedDirectory) {
 
 // processCacheEntry stores cached information for a single process.
 type processCacheEntry struct {
-	Tty         *stdio.TtyRelay
-	ContainerID string // If "" a host process otherwise a container process.
-	exitWg      sync.WaitGroup
-	exitCode    int
-}
-
-func newProcessCacheEntry(containerID string) *processCacheEntry {
-	return &processCacheEntry{ContainerID: containerID, exitCode: -1}
+	syncRoot sync.Mutex
+	Tty      *stdio.TtyRelay
+	exitWg   sync.WaitGroup // Signaled when the process itself has exited.
```

Comment:
nit: Put comment above `exitWg` to match pattern below. Plus, I'm not sure godoc recognizes comments that aren't above the thing being commented (although I could be wrong).

> The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration, with no intervening blank line.

Reply:
Fixed.